app/assets/stylesheets/templates/_ctc-faq.scss

```diff
@@ -0,0 +1,12 @@
+.ctc-faq {
+  .slab--stimulus-blue {
```

Comment:
This seems like a potentially odd way to target a specific slab. The modifier class of --stimulus-blue is primarily there to make it change color, but now we're also using it to override other things. In the hypothetical case where we add a second slab--stimulus-blue to this page, do we want it to act the same way? In the case that we want to change the color of the slab to purple or pink, do we want the div to lose these modifications?

I suggest potentially adding another modifier class to the slab you want to target in order to isolate the modifications OR take the entirely opposite approach and write a modifier class that affects grid directly to add flex/align center methods (which would likely be reuseable and may already exist?), and then use the spacing-bottom-0 modifier class on the h3 -- that will prevent about 10~ lines of new css generated in our resultant code.